lighthouse-core/report/html/report-styles.css

```diff
@@ -160,11 +160,6 @@
   display: none !important;
 }
 
-/* Edge doesn't recognize these, and the polyfill doesn't set block. https://github.com/javan/details-element-polyfill/issues/13 */
-details, summary {
-  display: block;
-}
```

Comment:
now [included in the polyfill](https://github.com/javan/details-element-polyfill/blob/5984dc6986720a739a89c484574f9a3e58537a58/dist/details-element-polyfill.js#L22)